src/quo/components/tags/collectible_tag/component_spec.cljs

```diff
@@ -0,0 +1,54 @@
+(ns quo.components.tags.collectible-tag.component-spec
+  (:require
+    [quo.components.tags.collectible-tag.view :as collectible-tag]
+    [test-helpers.component :as h]))
+
+(def collectible-name "Collectible")
+(def collectible-id "#123")
+(def ^:private theme :light)
+
+(defn get-test-data
+  [{:keys [options blur? size]}]
+  {:collectible-name    collectible-name
+   :collectible-id      collectible-id
+   :collectible-img-src {:uri "https://example.com/image.jpg"}
+   :options             options
+   :blur?               (or blur? false)
+   :size                (or size :size-24)
+   :theme               theme})
+
+(h/describe "Collectible_tag tests"
+  (h/test "Renders Default option"
+    (let [data (get-test-data {})]
+      (h/render-with-theme-provider [collectible-tag/view data] theme)
```

Comment:
`render-with-theme-provider` doesn't need a default theme anymore. If no theme is passed, a default `:light` is assume automatically.

Reply:
Done!